src/openMVG/types.hpp

```diff
@@ -28,19 +28,15 @@ typedef std::pair<IndexT,IndexT> Pair;
 typedef std::set<Pair> Pair_Set;
 typedef std::vector<Pair> Pair_Vec;
 
-#define OPENMVG_NO_UNORDERED_MAP 1
-
-#if defined OPENMVG_NO_UNORDERED_MAP
+#if defined OPENMVG_UNORDERED_MAP
```

Comment:
There is no reason to change that, so I don't see the point of adding more complexity in the build options.